hawkbit-mgmt-resource/src/main/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtRolloutMapper.java

```diff
@@ -105,6 +109,44 @@ static Rollout fromRequest(final EntityFactory entityFactory, final MgmtRolloutR
         return rollout;
     }
 
+    static RolloutGroup fromRequest(final EntityFactory entityFactory, final MgmtRolloutGroup restRequest) {
+        final RolloutGroup group = entityFactory.generateRolloutGroup();
+        group.setName(restRequest.getName());
+        group.setDescription(restRequest.getDescription());
+
+        if(restRequest.getTargetFilterQuery() != null) {
```

Comment:
By default the targetFilterQuery is an empty string. This avoids, that it is overwritten with null.